components/centraldashboard/public/components/namespace-selector_test.js

```diff
@@ -16,14 +16,14 @@ const TEMPLATE = `
 describe('Namespace Selector', () => {
     const namespaces = [
         {
-            user: 'testuser',
             namespace: 'default',
             role: 'contributor',
+            user: 'testuser',
```

Comment:
I dont think this change needs to happen. Maybe you should undo these changes with the order of the data to limit code diffs with upstream?